integration-cli/docker_api_containers_test.go

```diff
@@ -554,33 +554,6 @@ func (s *DockerAPISuite) TestContainerAPICreateEmptyConfig(c *testing.T) {
 	assert.ErrorContains(c, err, expected)
 }
 
-func (s *DockerAPISuite) TestContainerAPICreateMultipleNetworksConfig(c *testing.T) {
-	// Container creation must fail if client specified configurations for more than one network
-	config := container.Config{
```

Comment:
This test is still valid when using an older API version, but to test it, we would need to manually create a client with an older version, instead of the default `testEnv.APIClient()`.

Would be good to integrated that into the `TestCreateWithMultipleEndpointSettings`